backend/hyperglosae/src/views/lib/links.js

```diff
@@ -1,36 +1,55 @@
 exports.parseReference = parseReference = (reference) => {
-  let [id, fragment] = reference.split('#');
-  return {id, fragment};
-}
+  let [id, fragment] = reference.split("#");
+  return { id, fragment };
+};
 
 // Note: The result set includes also the document itself or its main document
-exports.getRelatedDocuments = ({isPartOf, links}) =>
+exports.getRelatedDocuments = ({ isPartOf, links }) =>
   new Set(
-    links.reduce((l, {object, subject}) => [...l, subject, object], [])
-      .filter(x => !!x)
+    links
+      .reduce((l, { object, subject }) => [...l, subject, object], [])
+      .filter((x) => !!x)
       .concat(isPartOf)
-      .map(x => parseReference(x).id)
+      .map((x) => parseReference(x).id)
   );
 
-exports.emitPassages = ({text, isPartOf, related}) => {
+exports.emitPassages = ({ text, isPartOf, related }) => {
```

Comment:
This line is not related to your contribution please don't include it in your commits.